<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mp3InfoWidget</class>
 <widget class="QWidget" name="Mp3InfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="filePath"/>
     </item>
     <item>
      <widget class="QPushButton" name="chooseBtn">
       <property name="text">
        <string>ChooseMp3</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="ID3InfoTable"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
